ci/authn-azure/get_system_assigned_identity.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash -ex
+
+decode_base64_url() {
+  local len=$((${#1} % 4))
+  local result="$1"
+  if [ $len -eq 2 ]; then result="$1"'=='
+  elif [ $len -eq 3 ]; then result="$1"'='
+  fi
+  echo "$result" | tr '_-' '/+' | openssl enc -d -base64
+}
+
+decode_jwt(){
+  decode_base64_url "$(echo -n $2 | cut -d "." -f $1)" | jq .
+}
+
+function bl_decode_jwt_payload(){
```

Comment:
why here you use function and other functions doesnt contain this?

Reply:
made the whole script clearer and easier to understand.